llvm/lib/Target/AArch64/AArch64ISelLowering.cpp

```diff
@@ -1454,8 +1454,12 @@ AArch64TargetLowering::AArch64TargetLowering(const TargetMachine &TM,
       setOperationAction(ISD::INSERT_SUBVECTOR, VT, Custom);
       setOperationAction(ISD::UINT_TO_FP, VT, Custom);
       setOperationAction(ISD::SINT_TO_FP, VT, Custom);
+      setOperationAction(ISD::STRICT_UINT_TO_FP, VT, Custom);
+      setOperationAction(ISD::STRICT_SINT_TO_FP, VT, Custom);
```

Comment:
Is there a requirement to change the behaviour of the `ISD::STRICT_` nodes?  If not then I'd rather drop them from this PR because SVE support for `strict_fp` is not really a thing yet so I'd rather not have partial support.  Plus I'm pretty sure Streaming SVE disables most all trapping math support anyway.

Reply:
Alright, removed the strict conversions for now :+1: